pkgs/applications/networking/cluster/kubecolor/default.nix

```diff
@@ -2,19 +2,23 @@
 
 buildGoModule rec {
   pname = "kubecolor";
-  version = "0.3.3";
+  version = "0.4.0";
 
   src = fetchFromGitHub {
     owner = pname;
     repo = pname;
```

Comment:
We should replace pname here with the kubecolor string literal with the next update